config.js.template

```diff
@@ -71,4 +71,15 @@ module.exports = {
   // to confirm that the client is connected to the right wallet. The primary
   // purpose of this parameter is to avoid operational mistakes and loss of funds.
   confirmFirstAddress: false,
+
+  // Optional config to enable plugins
+  // Hathor already provides some plugins, some of them are:
+  //   - debug: logs events to stdout
+  //   - ws: starts a websocket server on path /ws
+  // To enable a plugin, just add it's id on this parameter.
+  enabled_plugins: [],
+  // To add custom plugins you need to configure it on this parameter
+  // The key is the plugin id (for `enabled_plugins`) and the configuration requires
+  // a `file` property which is the filename of the plugin on the plugin directory
+  plugin_config: {},
```

Comment:
I have since removed the pluginDir since it has drifted from the original meaning.
The example I would rather add as a comment, since the template should (at least as i view it) be a "ready-to-go" basic configuration.